### DISCUSSION TITLE
how to set default view in mitmweb

### DISCUSSION THREAD

**Discussion by Stephen0910** [mitmproxy/mitmproxy#6320] [Q&A]
![image](https://github.com/mitmproxy/mitmproxy/assets/26613645/020f3285-8024-4927-9d41-8c25baf7d544)

I created a new view, but it's at the bottom.
May i move it up?
what's the better, it changes to the default view without selecting.
Thanks.

**Comment by mhils** [comment]
I don't think we have a setting for this in mitmweb yet, but contributions are welcome!

**Reply by Stephen0910** [reply]
thank you for your reply.
I solved this problem.
I rewrited the "custom_add" function to order the sort and defalt the view by "content_type"
![image](https://github.com/mitmproxy/mitmproxy/assets/26613645/8d1736d7-4281-4b0d-9542-c365648c0525)
![image](https://github.com/mitmproxy/mitmproxy/assets/26613645/58071866-1bc8-4a0d-bf4a-1e10c1107952)

still thank you very much.
